config/business_time.yml

```diff
@@ -0,0 +1,10 @@
+business_time:
+  beginning_of_workday: 12:00am
```

Comment:
Per product, we mostly want to exclude the weekends from our calculations. By extending the workday from 12:00am to 11:59pm.